### DISCUSSION TITLE
File is no longer null when uploading files

### DISCUSSION THREAD

**Discussion by oneinchman** [urql-graphql/urql#3170] [Q&A] [answered]
Hi!

I tried to update `@urql/core` to the latest version (4.0.4) and uninstall `@urql/exchange-multipart-fetch`. After the update, my API (which is compliant with [The GraphQL multipart fetch spec](https://github.com/jaydenseric/graphql-multipart-request-spec)) is throwing an internal server error. This is due to the operation JSON object now replaces `file` with an object instead of `null`. For example, `{"file: {"__key":"q6w2iwja1ta"}}` instead of `{"file": null}`. 

Is this intentional? If so, what is the recommended approach to support this for the API? Or can i force urql to set `file` as null somehow?

**Comment by JoviDeCroock** [comment, marked as answer]
Your issue got solved #3168 by #3169, why are you creating an additional discussion 😅 I assume the notification didn't correctly say it was solved as a PR?

**Reply by oneinchman** [reply]
Sorry, my bad. It didn't show up under my "issues" and I didn't receive any notifications for it so i thought it wasn't created. But all good now thanks.
